Game.Othello/Othello.cs

```diff
@@ -236,75 +176,164 @@ public OthelloState InitialState()
             return GenerateInitialOthelloState();
         }
 
-        (int up, int down, int left, int right) GetPotentialAction(int x, int y, OthelloState state)
+        PotentialAction GetPotentialAction (int x, int y, OthelloState state)
         {
             int up = 0;
             int down = 0;
             int left = 0;
             int right = 0;
+            int upLeft = 0;
+            int upRight = 0;
+            int downLeft = 0;
+            int downRight = 0;
             Field playersColor = (state.BlacksTurn) ? Field.Black : Field.White;
             Field opponentsColor = (state.BlacksTurn) ? Field.White : Field.Black;
```

Comment:
Tu można by dać jednego ifa, zamiast dwóch

Reply:
Ale jesteś pewien że to dobrze wygląda? Chyba w tym przypadku czytelność >> szybkość
`Field playersColor; 
            Field opponentsColor;
            if (state.BlacksTurn) 
            {
                playersColor = Field.Black;
                opponentsColor = Field.White;
            }
            else
            {
                playersColor = Field.White;
                opponentsColor = Field.Black;
            }`

Reply:
Ja wybieram szybkość :P

Reply:
(Field playersColor, Field opponentsColor) = 
                (state.BlacksTurn) ? (Field.Black, Field.White) : (Field.White, Field.Black);